driver.go

```diff
@@ -47,10 +47,18 @@ import (
 // Driver is exported to allow it to be used directly.
 type Driver struct{}
 
-// Open a Connection to the server.
-// See https://github.com/apache/calcite-avatica-go#dsn for more information
-// on how the DSN is formatted.
-func (a *Driver) Open(dsn string) (driver.Conn, error) {
+// Connector implements the driver.Connector interface
+type Connector struct {
+	additionalInfos   map[string]string
```

Comment:
Change this to `Info` to match the Java implementation.

Reply:
done